docs/sampleConfigurations/Templates/Introduction/C#/configuration.dsc.yaml

```diff
@@ -2,7 +2,7 @@
 
 ##########################################################################################################
 # This configuration will install the tools necessary to get started with C# development on Windows      #
-# Reference: https://learn.microsoft.com/en-us/visualstudio/get-started/csharp/tutorial-console          #
+# Reference: https://learn.microsoft.com/visualstudio/get-started/csharp/tutorial-console                #
```

Comment:
Is the extra whitespace needed?

Reply:
Yeah, it keeps the rectangle of #s a rectangle.